code/modules/holiday/spacemas.dm

```diff
@@ -488,6 +488,13 @@ var/static/list/santa_snacks = list(/obj/item/reagent_containers/food/drinks/egg
 		else
 			src.UpdateOverlays(null, "fire")
 
+	ephemeral //Disappears except on xmas
```

Comment:
I sorta feel like I'd rather have spawners that create the things if it is spacemas. The current way might get annoying in map editors if you see spacemas decorations all the time. But it's probably ok, I'm not a mapper afterall.

Reply:
A spawner approach could *potentially* work, if the spawner shared the icon state of what it spawns and the spawned xmas item inherited its offset and pixel rotation, but I'm not sure how complicated that would be to set up. As to having the things in the map editor all the time, it seems (at least to me) that it's a reasonable tradeoff for guaranteeing that the xmas version of the map is always up to date.

Reply:
Alright